robottelo/test.py

```diff
@@ -170,7 +171,10 @@ def setUpClass(cls):  # noqa
     def setUp(self):  # noqa
         """We do want a new browser instance for every test."""
         if self.driver_name.lower() == 'firefox':
-            self.browser = webdriver.Firefox()
+            self.browser = webdriver.Firefox(
+                firefox_binary=webdriver.firefox.firefox_binary.FirefoxBinary(
+                    self.driver_binary
+                ))
         elif self.driver_name.lower() == 'chrome':
             self.browser = webdriver.Chrome()
```

Comment:
The driver_binary don't work for Chrome too? If yes it is better to add it here using the Chrome way.